fusor-ember-cli/bower.json

```diff
@@ -13,8 +13,8 @@
     "jquery-csv": "google-code-export/jquery-csv#*",
     "loader.js": "ember-cli/loader.js#3.2.1",
     "qunit": "~1.18.0",
-    "bootstrap-sass": "~3.3.4",
-    "patternfly": "git://github.com/patternfly/patternfly#2.4.0",
+    "patternfly-sass": "2.9.0",
```

Comment:
I switched from `bootstrap-sass` to `patternfly-sass` and made the patternfly version consistent 2.9.0